pyflask/manageNeuroconv/manage_neuroconv.py

```diff
@@ -24,3 +26,15 @@ def get_all_interface_info() -> dict:
                 }
 
     return interface_info
+
+
+def get_combined_schema(interface_class_names: Optional[List[str]]) -> dict:
```

Comment:
```suggestion
def get_combined_schema(interface_class_names: List[str]) -> dict:
```

Now that there's no longer any default for `interface_class_name = None`, we can remove `Optional` from the annotation